tests/test_parser.py

```diff
@@ -303,3 +303,54 @@ def test_add_automatic_retrying(self):
             ast = SolidityAst(f'{contracts_root}/dev/buggy_10.sol', retry_num=0)
         except SolidityAstError:
             print("SolidityAstError is expected")
+
+
+    def test_event(self):
+        def sub_test(ast):
+            events = ast.events_in_contract_by_name('IPoolEvents')
+            self.assertEqual(15, len(events))
```

Comment:
To make the test more robust and readable, I suggest  creating expected event list and comparing it with the actual events list, for example replacing the code in `sub_test` with
```
assertEqual(set(expected_events_excluding_raw), set(actual_events_excluding_raw))
```